js/templates/modals/onboarding/onboarding.html

```diff
@@ -93,7 +93,7 @@
           <% if (ob.settingsErrors.currency) print(ob.formErrorTmpl({ errors: ob.settingsErrors.currency })) %>
           <select id="onboardingCurrency" name="localCurrency" class="clrSh2" data-model="settings">
             <% ob.currencyList.forEach((currency) => { %>
-            <option value="<%= currency.code %>" <% if (currency.code == ob.settings.localCurrency) print('selected') %>><%= currency.name %></option>
+            <option value="<%= currency.code %>" <% if (currency.code == ob.settings.localCurrency) print('selected') %>><%= currency.nameWithCode %></option>
```

Comment:
```
currency.nameWithCode
```

is nicer to use across multiple templates and easier to maintain if we need changes than:

```
<%= ob.polyT('currencyWithCode', { name: currency.name, code: currency.code }) %>
```

I definitely prefer the 'logic' being in one place, but I see your point for simplicity.

Reply:
@jjeffryes ^